setup.py

```diff
@@ -103,8 +103,9 @@
         # Until this issue is closed
         # https://github.com/googleapis/google-cloud-python/issues/10566
         "google-api-core[grpc] >= 1.31.5, <3.0.0dev,!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0",
-        "proto-plus >= 1.15.0",
-        "packaging >= 14.3",
+        "proto-plus >= 1.15.0, <2.0.0dev",
+        "protobuf >= 3.19.0, <4.0.0dev",
```

Comment:
Please also add `protobuf==3.19.0` to test the minimum supported version in `testing/constraints-3.6.txt`:

https://github.com/googleapis/python-aiplatform/blob/0036ab07004e0c9ae7806c4c2c25f22d5af4a978/testing/constraints-3.6.txt#L1-L8

Reply:
Thanks, added.